contracts/mainnet/connectors/compound/v3-rewards/main.sol

```diff
@@ -0,0 +1,75 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+/**
+ * @title Compound.
+ * @dev Rewards.
+ */
+
+import { TokenInterface } from "../../../common/interfaces.sol";
+import { Stores } from "../../../common/stores.sol";
+import { Helpers } from "./helpers.sol";
+import { Events } from "./events.sol";
+
+abstract contract CompoundV3RewardsResolver is Events, Helpers {
+	/**
+	 * @dev Claim rewards and interests accrued in supplied/borrowed base asset.
+	 * @notice Claim rewards and interests accrued.
+	 * @param market The address of the market.
+	 * @param account The account of which the rewards are to be claimed.
+	 * @param accrue Should accrue the rewards and interest before claiming.
+	 * @param setId ID stores the amount of rewards claimed.
+	 */
+	function claimRewards(
+		address market,
+		address account,
+		bool accrue,
```

Comment:
We should pass this as always true; no need to make it a param.

Reply:
@pradyuman-verma If this is true always, then `cometRewards.getRewardOwed(market, account).owed;` won't have updated rewards?

Reply:
No it doesn't matter whether we keep it as true or false, if we are calling `getRewardOwed` function as this function calls internally this function - https://github.com/compound-finance/comet/blob/main/contracts/Comet.sol#L425

Which increase user rewards, which is called by `claim` function if `accure` is true - https://github.com/compound-finance/comet/blob/main/contracts/CometRewards.sol#L153


So, we can keep it as false if we are calling the `getRewardOwed` function, as we already called the function to increase the accure amount of rewards.